src/rules/gamma_inverse/marginals.jl

```diff
@@ -0,0 +1,13 @@
+export marginalrule
```

Comment:
minor but these exports aren't necessary, they are exported automatically (we should remove it from all other places)

Reply:
I've removed both exports for rule and marginalrule